src/views/ModelPlan/ReadOnly/_components/ReadOnlySection/index.tsx

```diff
@@ -91,19 +91,21 @@ const ReadOnlySection = <
       <div
         className={`read-only-section read-only-section--${sectionName} margin-bottom-3`}
       >
-        <div className="display-flex flex-align-center">
-          <p className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line">
+        <div className="read-only-section--question">
+          <span className="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line display-block">
             {heading}
-          </p>
-          {config.questionTooltip && (
-            <Tooltip
-              className="margin-left-1"
-              label={config.questionTooltip}
-              position="right"
-            >
-              <Icon.Info className="text-base-light" />
-            </Tooltip>
-          )}
+            {config.questionTooltip && (
+              <span className="text-normal line-height-sans-2">
+                <Tooltip
+                  className="margin-left-1"
+                  label={config.questionTooltip}
+                  position="right"
+                >
+                  <Icon.Info className="text-base-light" />
+                </Tooltip>
+              </span>
+            )}
+          </span>
```

Comment:
Adjusted the code so that the `(i)` info icon is attached to the end of the question, instead of how it was floating around

![image](https://github.com/CMSgov/mint-app/assets/2349702/8f6f153e-6937-41e2-8465-9cfbe57baf05)